lib/open_food_network/subscription_payment_updater.rb

```diff
@@ -30,7 +30,8 @@ def create_payment
     end
 
     def card_required?
-      payment.payment_method.is_a? Spree::Gateway::StripeConnect
+      payment.payment_method.is_a?(Spree::Gateway::StripeConnect) ||
```

Comment:
`[Stripe::Gateway::StripeConnect, Spree::Gateway::StripeSCA].include? payment.payment_method.class` ?

Reply:
There are several of these checks that all look very similar. Maybe extracting it might be nice at some point?

Reply:
we will remove the existing stripe method soon so it will become what it was before...
but it would be good to have a central place where this check is done yes, I thought it would take a little bit to do it and I decided to skip it for now...